### PR TITLE
remove the unnecessary console log #5627

### LINKED ISSUE #5627: Remove unnecessary console logs
### Current Behavior
Currently layer5 website (https://layer5.io/) is having unnecessary console logs.

### Desired Behavior
These Console logs should be removed.

### Screenshots / Mockups
Screenshot from 2024-07-01 19-37-45

### Implementation
Simply remove the console logs.

### Note
I won't work on it. Kindly assign it to any new comer.

### Acceptance Tests

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
-  🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by r-kundan**
**Description**

This PR fixes #5627

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit d57359b6243e01ee09d97aa30457619b959b56f4 at: https://6683a92bc085f829f5172edc--layer5.netlify.app

File: src/components/Features/index.js [hunk 1]
```diff
@@ -32,7 +32,7 @@ const Features = (props) => {
         const maxY = rect.height - element.clientHeight - 50;
         const randomX = Math.floor(Math.random() * maxX);
         // const halfMaxY = maxY / 2;
-        console.log(maxX, maxY, "here");
+        // console.log(maxX, maxY, "here");
```
Comment (sudhanshutech) [inline src/components/Features/index.js:35]: please remove it , do not need to comment out

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit 5a64ad1eb43d2d1b06e6e4bfd596ba047c8f3abb at: https://6686bc41b0d22925983b8c18--layer5.netlify.app

File: src/components/Features/index.js (hunk 1 above)
Comment (r-kundan) [reply src/components/Features/index.js]: i have removed

**Comment by iArchitSharma** [comment]
Thank you @r-kundan

**Pull request merged**